projects/canopy/src/lib/table/docs/guide.stories.mdx

```diff
@@ -0,0 +1,238 @@
+      <th lg-table-head-cell>Book</th>
+      <th lg-table-head-cell>Published</th>
+    </tr>
+  </thead>
+  <tbody lg-table-body>
+    <tr lg-table-row>
+      <td lg-table-cell>Orhan Pamuk</td>
+      <td lg-table-cell>Strangeness in my Mind</td>
+      <td lg-table-cell>2016</td>
+    </tr>
+    <tr lg-table-row>
+      <td lg-table-cell>Albert Camus</td>
+      <td lg-table-cell>The Plague</td>
+      <td lg-table-cell>1947</td>
+    </tr>
+  </tbody>
+</table>
+```
+
+## How it works
+
+By default, most column data including text, date, postcode, phone number is left aligned. This helps make the data easily scannable, readable and comparable.
+
+The one exception is numeric data related to size (e.g count, percent). These numbers should be right aligned to help users identify number size.
+
+Column headers align to their column data.
+
+At mobile breakpoints the table layout changes to fit better in the viewport. The table adopts a 2 column layout, with the first column showing the header titles repeated on each row, and the second column showing the appropriate data points.
+
+### Table with expandable detail rows
+
+Expandable rows allow the user to evaluate additional information without losing context.
+
+This table adds the `LgTableRowToggleComponent`, and then an expandable detail row. This creates a table that allows the user to expand a row for more information. This can be seen on the [expandable detail story](./?path=/story/components-table-examples--expandable-table).
+
+```html
+<table lg-table>
+  <thead lg-table-head>
+    <tr lg-table-row>
+      <th lg-table-head-cell>Author</th>
+      <th lg-table-head-cell>Book</th>
+      <th lg-table-head-cell>Published</th>
+    </tr>
+  </thead>
+
+  <tbody lg-table-body>
+    <tr lg-table-row>
+      <td>
+        <lg-table-row-toggle
+          (click)="<function-to-handle-click>"
+          [isActive]="<logic-to-handle-toggle>"
+        >
+        </lg-table-row-toggle>
+      </td>
+    <td lg-table-cell>Orhan Pamuk</td>
+      <td lg-table-cell>Strangeness in my Mind</td>
+      <td lg-table-cell>2016</td>
+    </tr>
+    <tr lg-table-row [isHidden]="<logic-to-handle-row>">
+      <td lg-table-cell>
+        <lg-table-expanded-detail>
+          Detail content goes here
+        </lg-table-expanded-detail>
+      </td>
+    </tr>
+  </tbody>
+</table>
+```
+
+
+
+When scrolling, fix the header. This maintains context by keeping the labels in view at all times.
+
+### Table with long copy
+
+Sometimes a table can be used to display large amounts of copy, including buttons, links and headings.
+
+Note that each `LgTableCellComponent` has the stack Input set to true, which stacks the label and content on top of each other on small screens, instead of side by side. Also note the use of `<colgroup>` to control width of the columns on large screens. This can be seen on the [table with long copy story](./?path=/story/components-table-examples--with-long-copy-table).
```

Comment:
```suggestion
Note that each `LgTableCellComponent` has the `stack` Input set to true, which stacks the label and content on top of each other on small screens, instead of side by side. Also note the use of `<colgroup>` to control width of the columns on large screens. This can be seen on the [table with long copy story](./?path=/story/components-table-examples--with-long-copy-table).
```